### PR TITLE
[2.2]Operator workload Should be displayed in Rancher UI

### PR DESCRIPTION
**Pull request by orangedeng**
Related issue https://github.com/rancher/rancher/issues/14230

### CODE REVIEW

**Comment by alena1108** [comment]
Another related issue: https://github.com/rancher/rancher/issues/15804

**Comment by orangedeng** [comment]
Updated

File: pkg/api/store/workload/transform_store.go [hunk 1]
```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+var (
+	hideOwnerReferenceKinds = []string{"CronJob", "Deployment"}
```
Comment (alena1108) [inline pkg/api/store/workload/transform_store.go]: can we keep it in the map instead? This way we don't have to iterate over the slice, and do lookup better

**Comment by alena1108** [comment]
https://github.com/rancher/rancher/issues/16277

**Comment by alena1108** [comment]
LGTM

**Pull request merged**